wannierberri/__main.py

```diff
@@ -149,6 +150,8 @@ def integrate(system,grid,Efermi=None,omega=None, Ef0=0,
         evaluate only symmetry-irreducible K-points
     symmetrize : bool
         symmetrize the result (always `True` if `use_irred_kpt == True`)
+    print_Kpoints : bool
+        print the list of K points (default: True)
```

Comment:
There is generally no need to write the defaults in the docstrings, because when sphinx generates documentation it takes defaults from the method definition. Thus it creates a risk to make the docstring to become in consistant with the actual definition of the defaults.